### PR TITLE
Upgrade supported Python versions and CI config

### PR DESCRIPTION
**Pull request by Gallaecio**

### CODE REVIEW

File: sklearn_crfsuite/estimator.py [hunk 1]
```diff
@@ -263,12 +260,14 @@ def __init__(self,
         self.averaging = averaging
         self.variance = variance
         self.gamma = gamma
+        self.model_filename = model_filename
+        self.keep_tempfiles = keep_tempfiles
```
Comment (kmike) [inline sklearn_crfsuite/estimator.py:264]: Why is this needed?

Comment (Gallaecio) [reply sklearn_crfsuite/estimator.py:264]: https://github.com/TeamHG-Memex/sklearn-crfsuite/issues/60
https://github.com/TeamHG-Memex/sklearn-crfsuite/pull/74

**Review by kmike — APPROVED** [review]

**Review by wRAR — APPROVED** [review]
